keylime/policy/create_runtime_policy.py

```diff
@@ -972,7 +972,7 @@ def create_runtime_policy(args: argparse.Namespace) -> Optional[RuntimePolicyTyp
                 )
                 abort = True
         else:
-            if a not in algorithms.Hash:
+            if a not in [h.value for h in algorithms.Hash]:
```

Comment:
This works, but I wonder if something like `if a not in set(algorithms.Hash)` would work and be more readable?